install.sh

```diff
@@ -29,41 +29,43 @@ BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 echo "# Creating new environment for Macrel"
 mkdir -p envs
-conda create --yes -p $BASEDIR/envs/Macrel_env python=$PYTHON_VERSION
+conda create --yes -p $BASEDIR/envs/Macrel_env python=$PYTHON_VERSION ## Fixed 
+source activate $BASEDIR/envs/Macrel_env
 conda config --env --add channels defaults
 conda config --env --add channels bioconda
 conda config --env --add channels conda-forge
 
+echo "# Installing packages with MAMBA"
 if ! which mamba > /dev/null; then
     CONDA_INSTALL_CMD=conda
+    conda install -y \
+              ngless \
+              megahit \
+              paladin \
+              pandas \
+              scikit-learn \
+              atomicwrites \
+              tzlocal \
+              --quiet
 else
-    echo "# Installing packages with MAMBA"
-    CONDA_INSTALL_CMD=mamba
+    mamba install -y \
+              ngless \
+              megahit \
+              paladin \
+              pandas \
+              scikit-learn \
+              atomicwrites \
+              rpy2 \
+              tzlocal \
+              --quiet
 fi
-${CONDA_INSTALL_CMD} install -y \
-          --prefix $BASEDIR/envs/Macrel_env \
-          ngless \
-          megahit \
-          paladin \
-          pandas \
-          scikit-learn \
-          atomicwrites \
-          tzlocal \
-          make \
-          c-compiler
-
-# only activate AFTER install all the packages to get the right environmental variables
-source activate $BASEDIR/envs/Macrel_env
 
-# env forwards the current environment and sets some extra variables needed by
-# ./recipe/build.sh
 env \
         PYTHON=$(which python) \
         SRC_DIR=$PWD \
         PREFIX=$CONDA_PREFIX \
-        ./recipe/build.sh
-
-
+        sh ./recipe/build.sh
```

Comment:
The file uses `#!/usr/bin/bash` not `sh`. Not sure if it matters, but should be consistent

Reply:
it was corrected now